vmalert: do not send message to alertmanager when alert has no label … (#6823)
VictoriaMetrics/VictoriaMetrics: docs/changelog/CHANGELOG.md

## tip

> * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert): do not send notifications to alertmanager if alert has no label-value pairs left after applying `alert_relabel_configs` in [notifier config](https://docs.victoriametrics.com/vmalert/#notifier-configuration-file), since alertmanager would refuse such an alert anyway.

Reviewer: ```suggestion
* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert): do not send notifications without labels to Alertmanager. Such notifications are rejected by Alertmanager anyway. Before, vmalert could send alert notifications even if no label-value pairs left after applying `alert_relabel_configs` from [notifier config](https://docs.victoriametrics.com/vmalert/#notifier-configuration-file). 
```

* BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent): fixes `proxy_url` authorization for scrape targets. Previously proxy authorization configuration was ignored for `https` targets. See [this](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6771) issue for details.
* BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent/) fix service discovery of Azure Virtual Machines for response contains `nextLink`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6784).
* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert): respect HTTP headers defined in [notifier configuration file](https://docs.victoriametrics.com/vmalert/#notifier-configuration-file) for each request to notifiers. Previously, this param was ignored by mistake.

> * BUGFIX: [vmalert](./vmalert.md): do not send message to alertmanager when alert has no label pair, alertmanager would refuse such an alert and report error. It can be achieved by adding `alert_relabel_configs` in [notifier config](https://docs.victoriametrics.com/vmalert/#notifier-configuration-file).

Reviewer: ```suggestion
* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert): do not send notifications to alertmanager if alert has no label-value pairs left after applying `alert_relabel_configs` in [notifier config](https://docs.victoriametrics.com/vmalert/#notifier-configuration-file).  Alertmanager would refuse notifications without label-value pairs anyway.
```

Please always use absolute links
